jameda_interface_config.js

```diff
@@ -79,9 +79,9 @@ var interfaceConfig = {
 
     SETTINGS_SECTIONS: [
         'devices',
-        'profile',
         'language'
 
+        // 'profile',
```

Comment:
Please remove this line if it's not necessary.

Reply:
The commented attributes in SETTINGS_SECTIONS I have purposely not deleted due to I believe we might need these back in the near future. However, I can delete and refer to jitsi original repo as well. ;)